packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -556,16 +407,16 @@ export function getSupportedNetworks(chainId = 0, includeTestnets = false) {
     ChainId.ArbitrumGoerli,
     ChainId.Sepolia,
     ChainId.ArbitrumSepolia,
-    ChainId.XaiTestnet,
     ChainId.StylusTestnet,
+    ...Object.keys(orbitTestnets).map(Number),
```

Comment:
Instead of doing `Object.keys` on the object let's just augment `getOrbitChains` so you can choose whether to include testnet/mainnet (both should default to true)

```function getOrbitChains(options: { mainnet: boolean, testnet: boolean })```